src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-sidebar/dot-edit-layout-sidebar.component.ts

```diff
@@ -74,7 +76,7 @@ export class DotEditLayoutSidebarComponent implements OnInit, ControlValueAccess
      * @param {DotContainerColumnBox[]} containers
      * @memberof DotEditLayoutSidebarComponent
      */
-    updateContainers(containers: DotContainerColumnBox[]): void {
+    updateContainers(containers?: DotContainerColumnBox[]): void {
```

Comment:
Does hhis method needs to be renamed and redocumented? @freddyucv @hmoreras let's talk about this.

Reply:
mmm, this method was originally called liked that to propagate/update the containers to the parent, now it's been reused by 2 children to propagate any change than happened on the children components.

Maybe rename it to updateModel(). Thoughts?